src/Setup/DevDivVsix/RoslynVisualStudioDiagnosticsWindow/Roslyn.VisualStudio.Diagnostics.Window.Resources.AllLangsNoEnu.swr

```diff
@@ -0,0 +1,2 @@
+folder "TBD"
+  file source=$(OutputPath)\Vsix\Roslyn.VisualStudio.DiagnosticsWindow\$(Culture)\Microsoft.Build.Tasks.CodeAnalysis.resources.dll
```

Comment:
@tmeschter 
I can't find this assembly under the output/Dlls or output/Exes, can I just reference it under Vsix folder?
Also I find that a lot of resources assemblies are seems not generated by the compilation of Rolsyn, should I also included them?